tf_big/python/ops/big_ops_test.py

```diff
@@ -1,6 +1,10 @@
+import unittest
+from absl.testing import parameterized
+
 import numpy as np
 import tensorflow as tf
-from tensorflow.python.platform import test
+
+from tf_encrypted.test import tf_execution_context
```

Comment:
TFE isn't declared as a dependency of this package as far as I can see, won't that cause problems?

Reply:
moreover, if `tf-big` is eventually a dependency of `tf-encrypted` (or `tf-encrypted-primitives`), I'm a bit concerned about setting precedent for a dependency mess

Reply:
yeah good point. think I originally planned on simply copying the code in from TFE once the PR there had landed. perhaps we should stick to that strategy.

Reply:
done